tests/integration/workflows/rust_cargo/test_rust_cargo.py

```diff
@@ -0,0 +1,64 @@
+import os
+import shutil
+import tempfile
+
+from unittest import TestCase
+
+from aws_lambda_builders.builder import LambdaBuilder
+from aws_lambda_builders.exceptions import WorkflowFailedError
+
+
+class TestRustCargo(TestCase):
```

Comment:
I know Rust won't compile if something is wrong. This means, we will want to ensure that the build logs and error messages are properly surfaced back to customers. I think adding some integ test around this will help us know:
1. We are providing enough info back to customers about why their build failed.
2. Help us not regress these actions accidentally in the future.

Would you be able to add a couple tests in this area?

Reply:
Rusts compilation errors are family help at understanding why a build failed and what to do about it. Rather than reinvent that it's exposed directly through the popen stdout/err hooks

I can see if there's a way to structure a passing test for a failing build

Reply:
Added an integration test for a failing build